capsules/extra/src/ieee802154/driver.rs

```diff
@@ -1070,6 +1070,7 @@ impl<'a, M: device::MacDevice<'a>> device::TxClient for RadioDriver<'a, M> {
 
 /// Encode two PAN IDs into a single usize.
 #[inline]
+#[allow(dead_code)]
```

Comment:
This doesn't need to be here, right?

Reply:
I modified the arguments passed with the scheduled upcall to no longer include the pan so this function is no longer called. I figured we'd want to keep this though since it may still be useful in the future?

Reply:
I see. I thought that was for debugging or something.

I could go either way on it. Probably you're right to keep it.

Reply:
Let's decide if we want this or not. If we do, then let's have two upcalls, one that is the same as today, and one that only provides lqi.

If not, let's delete these functions. Future readers aren't going to understand what these functions are for.

Reply:
We discussed this Monday in the network-wg and decided to remove these functions. Will update PR now.